docs/stencil-docs/storefront-customization/using-custom-fonts-and-icons.md

```diff
@@ -33,8 +33,9 @@ There are two paths to take for using custom fonts. They are as follows:
 
 ### Google Fonts
 
-[Google Fonts](https://fonts.google.com/) are a collection of open source fonts available for use. The base Cornerstone theme uses Karla and Montserrat. Google Fonts come included in the Cornerstone theme.  In `config.json` update each place you want the change with the name of the new Google Font.
-Make sure it follows of the format `Google_FontName_Weight`.
+[Google Fonts](https://fonts.google.com/) are a collection of open source fonts available for use. The base Cornerstone theme uses Karla and Montserrat. Google Fonts come included in the Cornerstone theme.  

```

Comment:
[Google Fonts](https://fonts.google.com/) are a collection of open-source fonts available for use. The base Cornerstone theme uses Karla and Montserrat. Google Fonts come included in the Cornerstone theme.